MASFoundation/Classes/models/MASConfiguration.m

```diff
@@ -889,6 +891,22 @@ - (BOOL)enabledTrustedPublicPKI
     return [mobileInfo[MASMobileEnbaledTrustedPublicPKI] boolValue];
 }
 
+
+- (BOOL)allowSSLPinning
+{
+    NSDictionary *magInfo = _configurationInfo_[MASMAGConfigurationKey];
+    NSDictionary *mobileInfo = magInfo[MASMobileConfigurationKey];
+    
+    //return [mobileInfo[MASMobileAllowSSLPinningConfigurationKey] boolValue];
+    
+    //
+    // TODO: Uncomment the above return statement.
```

Comment:
Do we have todo anything here?

Reply:
This was included to get msso_config.json attribute. Since that will need MAG changes and will be delayed. This stub will have to be updated once that attribute is available in msso_config.json